Sources/MarathonCore/ScriptManager.swift

```diff
@@ -259,6 +271,25 @@ internal final class ScriptManager {
 
         return scriptFolder
     }
+    
+    @discardableResult private func createTestsFolderIfNeeded(for script: Script, file: File) throws -> Folder {
+        if !script.folder.containsFile(named: "OriginalTestsFile") {
+            try script.folder.createSymlink(to: file.path, at: "OriginalTestsFile", printer: printer)
+        }
+        
+        let testsFolder = try script.folder.createSubfolderIfNeeded(withName: "Tests")
+        try testsFolder.empty()
+        
+        let testsModuleFolder = try testsFolder.createSubfolderIfNeeded(withName: script.name + "Tests")
+        try testsModuleFolder.empty()
```

Comment:
No need to empty here, you are already emptying the parent folder on line `281`.